embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.kt

```diff
@@ -379,7 +382,7 @@ internal class EmbraceImpl @JvmOverloads constructor(
         }
     }
 
-    private fun useLegacyResurrection(): Boolean {
-        return configService?.autoDataCaptureBehavior?.isV2StorageEnabled() != true
+    private fun useLegacyResurrection(cfgService: ConfigService?): Boolean {
+        return cfgService?.autoDataCaptureBehavior?.isV2StorageEnabled() != true
```

Comment:
Fixes a subtle bug where `configService` returns null because the SDK isn't initialized yet

Reply:
Good catch!